packages/components/src/radio-group/types.ts

```diff
@@ -0,0 +1,39 @@
+export type RadioGroupProps = {
+	/**
+	 * Labels the value of the selected `Radio` element.
```

Comment:
This doesn't seem accurate, maybe something this instead?

```suggestion
	 * Accessible label for the radio group
```